### PR TITLE
Minor optimization

### PR DESCRIPTION
**Pull request by anisimkov**
Do not parse already parsed info.
Use new Split_Path routine in HTTP/1 and HTTP/2.

### CODE REVIEW

File: src/core/aws-server-protocol_handler_v2.adb [hunk 1]
```diff
@@ -574,11 +574,24 @@ procedure Protocol_Handler_V2 (LA : in out Line_Attribute_Record) is
 
       --  And set the request information using an HTTP/1 request line format
 
-      AWS.Server.HTTP_Utils.Parse_Request_Line
-        (M.Headers.Get (Messages.Method_Token) & ' '
-         & M.Headers.Get (Messages.Path2_Token)
-         & " HTTP_2",
-         LA.Stat);
+      declare
+         Path : constant String := M.Headers.Get (Messages.Path2_Token);
```
Comment (TurboGit) [inline src/core/aws-server-protocol_handler_v2.adb:578]: : should be aligned if line too long split after :=

File: src/core/aws-server-http_utils.adb [hunk 2]
```diff
@@ -2278,4 +2209,35 @@ package body AWS.Server.HTTP_Utils is
                  and then not Utils.Match (Connection, "keep-alive"));
    end Set_Close_Status;
 
+   ----------------
+   -- Split_Path --
+   ----------------
+
+   procedure Split_Path (Path : String; Path_Last, Query_First : out Positive)
+   is
```
Comment (TurboGit) [inline src/core/aws-server-http_utils.adb:2217]: Minor style update:

```
 procedure Split_Path 
  (Path                   : String; 
   Path_Last, Query_First : out Positive)
 is
```

**Review by TurboGit — CHANGES_REQUESTED** [review]
Some style fixes.

File: src/core/aws-server-http_utils.adb (hunk 2 above)
Comment (anisimkov) [reply src/core/aws-server-http_utils.adb:2219]: Done,

**Review by TurboGit — APPROVED** [review]
All good, thanks !

**Pull request merged**